docs/source/notebooks/dap_subset.ipynb

```diff
@@ -1881,17 +1882,17 @@
    "execution_count": 7,
    "metadata": {
     "execution": {
-     "iopub.execute_input": "2022-01-24T17:53:23.320348Z",
-     "iopub.status.busy": "2022-01-24T17:53:23.319795Z",
-     "iopub.status.idle": "2022-01-24T17:53:23.322634Z",
-     "shell.execute_reply": "2022-01-24T17:53:23.323051Z"
+     "iopub.execute_input": "2022-02-04T18:10:46.157130Z",
+     "iopub.status.busy": "2022-02-04T18:10:46.154311Z",
+     "iopub.status.idle": "2022-02-04T18:10:46.164347Z",
+     "shell.execute_reply": "2022-02-04T18:10:46.166583Z"
     }
    },
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "{'lat': 1, 'lon': 2, 'time': slice(5040, 6840, None)}"
+       "{'lat': array(1), 'lon': array(2), 'time': slice(5040, 6840, None)}"
```

Comment:
@aulemahal Missed ignore output on this one as well !  I'll do it so I can try this tags thing.